libs/application/templates/driving-license/src/fields/HealthDeclaration.tsx

```diff
@@ -46,6 +46,7 @@ function HealthDeclaration({
           <RadioController
             id={field.id}
             split="1/2"
+            largeButtons={false}
```

Comment:
Based on the application design system, it should always be true cf: https://www.figma.com/file/ine6cGn7cnrJJK43fzUZTF/Templates-%2F-h%C3%B6nnunarkerfi-fyrir-ums%C3%B3knir?node-id=2%3A6045

Reply:
i see.. it looked super weird when i didn't change it after your change to the default - perhaps i was missing some props.

so this is based on a design that needs to be updated then.

Reply:
Somebody pointed out that those are not very suitable for yes/no questions.. you are talking about these, correct?

![image](https://user-images.githubusercontent.com/61151/119691868-75488680-be3a-11eb-9e44-8eeebd675e52.png)

Reply:
yes, this feels like something for the designers to argue about. I'll tell Orvar and he can update the design or bring it up with the other designers.

Thanks for pointing this out, but Arndis, my skilastjóri doesn't want to have this changed right now.